src/evict/evict_inline.h

```diff
@@ -76,20 +93,19 @@ __wt_evict_read_gen_new(WT_SESSION_IMPL *session, WT_PAGE *page)
 }
 
 /*
- * __wt_evict_cache_stuck --
- *     Indicate if the cache is stuck (i.e., eviction not making progress).
+ * __wt_readgen_evict_soon --
+ *     Return whether a page's read generation makes it eligible for immediate eviction. Read
+ *     generations reserve a range of low numbers for special meanings and currently - with the
+ *     exception of the generation not being set - these indicate the page may be evicted
+ *     immediately.
  */
 static WT_INLINE bool
-__wt_evict_cache_stuck(WT_SESSION_IMPL *session)
+__wt_readgen_evict_soon(uint64_t *readgen)
```

Comment:
It is removed from `btree_inline.h` in a later ([11ca8bb](https://github.com/wiredtiger/wiredtiger/pull/11087/commits/11ca8bb682558dfbbcc188aa55d6f4e49986ef2a))commit. Sorry about that (this should have been the first commit)